api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/summary/Helpers.kt

```diff
@@ -0,0 +1,138 @@
+package org.jetbrains.kotlinx.dl.api.core.summary
+
+import mu.KLogging
+import org.jetbrains.kotlinx.dl.api.core.TrainableModel
+import org.slf4j.Logger
+import java.io.PrintStream
+
+private data class SummaryRow(
+    val layerName: String,
+    val outputShape: String,
+    val paramsCount: String,
+    val inboundLayers: List<String>
+)
+
+/**
+ * Formats model summary
+ * @param layerNameColumnName title of the column with layer names
+ * @param outputShapeColumnName title of the column with layer output shapes
```

Comment:
Please add brackets to the parameter names [layerNameColumnName]

Reply:
done